### ISSUE TITLE
Declaration of @noreturn function shows with additional characters at the end

### ISSUE THREAD

**Issue by emaloney** [realm/jazzy#145]
jazzy 0.0.20 seems to include extra characters at the end of the declaration section for functions declared as `@noreturn`.

In the following screenshot, a portion of the text from the function implementation is visible in the declaration:

![screen shot 2015-01-07 at 4 17 25 pm](https://cloud.githubusercontent.com/assets/3768925/5653505/cd115670-9688-11e4-8dc2-1378a91a7527.png)

The documentation and function declaration are as follows:

```swift
/**
To be used from within unimplemented functions. A function might not be
implemented because development is still in progress, or because a subclass
was expected to override the function to provide an implementation. This
function is intended to be called with zero arguments.
*/
@noreturn func errorNotImplemented(function: String = __FUNCTION__)
{
    fatalError("Function not implemented: \(function)")
}
```

It looks to be 8 extra characters: "`{[return][space][space][space][space]fa`"

Not sure if it's relevant, but this function is declared in the global scope.

**Comment by jpsim** [comment]
thanks for filing this, @emaloney. Declaration parsing is done by SourceKitten, outside of SourceKit itself, and that code isn't very robust at the moment. We're working on a pretty big SourceKitten overhaul at the moment and we'll make sure this is fixed by SourceKitten 0.3.0. This shouldn't take too long, hopefully.

**Comment by jpsim** [comment]
This is fixed as of jazzy 0.1.5. Thanks for reporting, @emaloney!
